functional_tests/step_helpers/release_page_helpers.py

```diff
@@ -0,0 +1,287 @@
+    change_to_release_date_section.get_by_label("Reason for change*").fill("Updated due to data availability")
+
+
+def add_release_date_text(context: Context) -> None:
+    fill_locator(context, RELEASE_DATE_TEXT, "March 2025 to August 2025")
+
+
+def add_next_release_date_text(context: Context) -> None:
+    fill_locator(context, NEXT_RELEASE_DATE_TEXT, "To be confirmed")
+
+
+def add_related_link(context: Context) -> None:
+    context.page.locator("#panel-child-content-related_links-content").get_by_role(
+        "button", name="Insert a block"
+    ).first.click()
+    choose_page_link(context.page, page_name="Home")
+
+
+def add_pre_release_access_info(context: Context) -> None:
+    add_basic_table_block_under_pre_release_access(context)
+    add_description_block_under_pre_release_access(context, index=1)
+
+
+def add_invalid_release_date_text(context: Context) -> None:
+    fill_locator(context, RELEASE_DATE_TEXT, "Invalid 5555")
+
+
+def add_invalid_next_release_date_text(context: Context) -> None:
+    fill_locator(context, NEXT_RELEASE_DATE_TEXT, "Invalid 5555")
+
+
+def add_next_release_date_to_be_earlier_than_release_date(context: Context) -> None:
+    fill_locator(context, RELEASE_DATE, "2025-12-25")
+    fill_locator(context, NEXT_RELEASE_DATE, "2024-12-25")
+
+
+def add_both_next_release_date_and_text(context: Context) -> None:
+    fill_locator(context, NEXT_RELEASE_DATE, "2025-12-25")
+    fill_locator(context, NEXT_RELEASE_DATE_TEXT, "December 2024")
+
+
+def expect_related_links(context: Context) -> None:
+    expect(context.page.get_by_role("heading", name="You might also be interested")).to_be_visible()
+    expect(context.page.locator("#links").get_by_role("link", name="Home")).to_be_visible()
+
+
+def expect_either_release_date_or_next_release_date(context: Context) -> None:
+    expect(
+        context.page.locator(
+            "#panel-child-content-child-metadata-child-panel1-child-next_release_date-errors"
+        ).get_by_text("Please enter the next release date or the next release text, not both.")
+    ).to_be_visible()
+    expect(
+        context.page.locator(
+            "#panel-child-content-child-metadata-child-panel1-child-next_release_date_text-errors"
+        ).get_by_text("Please enter the next release date or the next release text, not both.")
+    ).to_be_visible()
+
+
+def expect_changes_to_release_date(context: Context) -> None:
+    preview_texts: list[str] = [
+        "Previous date",
+        "25 December 2024 9:30am",
+        "Reason for change",
+        "Updated due to data availability",
+    ]
+    expect_text(context, key=None, texts=preview_texts)
+    expect(context.page.get_by_role("link", name="Changes to this release date")).to_be_visible()
+    expect(context.page.get_by_role("heading", name="Changes to this release date")).to_be_visible()
+
+
+def add_multiple_descriptions(context: Context) -> None:
```

Comment:
You can replace the value for the key with a lambda that performs the same action inside the handlers dictionary